### PR TITLE
Updated docs a bit to help guide through first run experience.

### PR DESCRIPTION
**Pull request by hannesne**

### CODE REVIEW

File: docs/running-deploying.md [hunk 1]
```diff
@@ -28,13 +28,22 @@ make install-requirements
...
-```
+1. Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this. The first time you run this, you should be sure to set at least the following environment variables.
+
+    ```python
+    SIMULATOR_API_KEY=my-test-key
+    TEST_OPENAI_ENDPOINT=http://localhost:8000/
+    TEST_OPENAI_KEY=my-test-key
+    ```  
+
+2. Start (or restart) the simulator by running the following command in your terminal from the repository root directory:
```
Comment (martinpeck) [inline docs/running-deploying.md:39]: What do you mean by restart? If you run `make run-simulated-api` it doesn't return until the service is killed, and if I open another terminal and run `make run-simulated-api` a second time it won't work.

I assume you're saying "or stop, and then restart it, if it's already running" but I think we should assume that anyone going through this "first run" experience won't have started it yet. You can't run the `make` command if it's already running, so anyone following these 4 steps will have to stop it to follow step 2.

File: docs/running-deploying.md [hunk 2]
```diff
@@ -28,13 +28,22 @@ make install-requirements
...
 ## Running the Simulator Locally
 
-Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./docs/config.md) for details on how to do this.
-
-To run the simulated API, use the following command from the repository root directory:
-
-```console
-make run-simulated-api
-```
+1. Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this. The first time you run this, you should be sure to set at least the following environment variables.
```
Comment (martinpeck) [inline docs/running-deploying.md:31]:
```suggestion
1. Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this. 

The minimum set of environment variables you'll need in your `.env` file to run the simulator locally are as follows:
```

File: docs/running-deploying.md [hunk 3]
```diff
@@ -28,13 +28,22 @@ make install-requirements
...
+
+    ```python
+    SIMULATOR_API_KEY=my-test-key
+    TEST_OPENAI_ENDPOINT=http://localhost:8000/
+    TEST_OPENAI_KEY=my-test-key
```
Comment (martinpeck) [inline docs/running-deploying.md:36]:
```suggestion
    TEST_OPENAI_KEY=my-test-key
    TEST_OPENAI_DEPLOYMENT=gpt-3.5-turbo-0613
```

File: docs/running-deploying.md (hunk 1 above)
Comment (hannesne) [reply docs/running-deploying.md:39]: Updated with another line regarding restarting the server to pick up config changes.

File: docs/running-deploying.md [hunk 4]
```diff
@@ -28,13 +28,26 @@ make install-requirements
...
+    ```  
+
+2. Start the simulator by running the following command in your terminal from the repository root directory:
+  
+    ```console
+    make run-simulated-api
+    ```
+
+    Kill the process and run the command again to restart the simulator whenever you make changes to the config.
+3. Now open the [test-aoai.http](test-aoai.http) file, and send the first POST request.
```
Comment (stuartleeks) [inline docs/running-deploying.md:49]: This link doesn't resolve - based on the env var, I assume this is intended to work with the chat completions http file?

```suggestion
3. Now open the [http/chat-completions.http](../http/chat-completions.http) file, and send the first POST request.
```

Comment (hannesne) [reply docs/running-deploying.md]: fixed

**Review by stuartleeks — APPROVED** [review]
LGTM

**Pull request merged**